src/app/editor/viewer/lib/sketch-canvas.component.ts

```diff
@@ -119,10 +119,11 @@ export class SketchCanvasComponent implements OnInit, AfterViewInit {
     const current = this.canvasRef.nativeElement.getBoundingClientRect();
     this.positionX = current.left - 100;
     this.positionY = current.top;
+    // debugger;
   }
 
   formatTransformStyle(existingTransformStyle: string, zoomLevel) {
-    const scaleStyleRegex = /(\([ ]?[\d]+(\.[\d]+)?[ ]?(,[ ]?[\d]+(\.[\d]+)?[ ]?)?\))/gmi;
+    const scaleStyleRegex = /(\([ ]?[\d]+(\.[\d]+)?[ ]?(,[ ]?[\d]+(\.[\d]+)?[ ]?)?\))/gim;
```

Comment:
Just checking but should this be `gmi` or `gim`?

Reply:
these are just flags so there are interchangeable 🙂